cli_bot.py

```diff
@@ -54,9 +55,34 @@ def __init__(self, address_book: AddressBook):
             "birthdays": self.birthdays,
             "add-address": self.add_address,
             "add-email": self.add_email,
+            "help": self.help,
         }
         self._address_book = address_book
 
+    def help(self, *args: str) -> str:
+        """Outputs a help message for user."""
+        command_output = ""
+        if len(args) > 1:
+            command_output += (
+                "Warning: Command optionally expect one argument - command name. "
+                f"Received: {' '.join(args)}\n"
+            )
+
+        search_command = args[0] if args else None
+
+        if search_command:
+            if search_command in self.supported_commands:
+                command_output += f"Command '{search_command}' help: "
+                command_output += f"\n{self.supported_commands[search_command].__doc__.split('Raises:')[0].strip()}"
```

Comment:
This looks a bit dangerous, there can be commands that doesn't raise anything (For example print_all, it has warnings only). Is there a better way?

Reply:
Does not split return a full string in this case?

Reply:
Ack

Reply:
Btw, can we split by 'Returns:' instead ? It doesn't seem like it's a meaningful info for a user to know that handler function will return a Command output

Reply:
make sense